src/dcrypto/v1/v1_test.go

```diff
@@ -92,3 +94,37 @@ func TestRoundTrip(t *testing.T) {
 		}
 	}
 }
+
+func TestHash(t *testing.T) {
+	sizes := []int{24, 1024, 15872, 16364, 16384, 16394, 16896, 66560}
+	for _, size := range sizes {
+		h := md5.New()
```

Comment:
`I thought drive's hash of the remote files were md5 so just used that but I can use something else just so people don't see it and assume the worst.`
Ahh I see what you are saying. Although the files are md5'd by default, with no password and no ciphers.

`I forgot to ask if you think you'll need this logic? I was trying to figure out how to solve the issue of determining what has changed so did this.`
Actually for starters if the file is marked as encrypted at rest on Google Drive, we can select its `md5sum`(always provided by Google Drive for any file), then hash the one locally and send that to an md5sum cipher locally to see if they changed. I think users will understand that this doesn't come for free. Also we can perform these checks after filtering through some criteria like modTime changes, metaData changes.
I honestly don't haven't solidly thought of a way to accomplish this but I think we can keep the code and then later figure it out.